wangle/acceptor/ManagedConnection.h

```diff
@@ -50,7 +50,7 @@ class ManagedConnection:
   };
 
   // HHWheelTimer::Callback API (left for subclasses to implement).
-  void timeoutExpired() noexcept override = 0;
+  virtual void timeoutExpired() noexcept = 0;
```

Comment:
But this is an override:

https://github.com/facebook/folly/blob/v2020.11.02.00/folly/io/async/AsyncTimeout.h#L79-L82